.github/workflows/docker_repo_tests.yaml

```diff
@@ -0,0 +1,115 @@
+name: repo_tests
+on: [workflow_dispatch, workflow_call]
+
+jobs:
+  repo_tests:
+    name: ${{ matrix.repo }} ${{ matrix.description }}
+    runs-on: ubuntu-latest
+    container: catthehacker/ubuntu:runner-20.04
```

Comment:
This is a docker image ([link](https://github.com/catthehacker/docker_images/pkgs/container/ubuntu)) that mimics the github runner image

Reply:
add this as a comment inline